doc/release/release_dev.rst

```diff
@@ -12,6 +12,8 @@ For more information, examples, and documentation, please visit our website:
 https://scikit-image.org
 
 
+This release of scikit-image drops support for Python 3.5.
+This release of scikit-image officially supports Python 3.6 and 3.7.
```

Comment:
Perhaps we should state the policy going forward?

Reply:
I thought release notes were about facts, not decision process. Maybe it can go somewhere else?

Reply:
When the API or supported versions change, affected users typically want to know why. It's the reason why we "link" to the PRs, I think. You could add a "see #4102 for more info", but you can explain the NEP in about the same space.

Reply:
How about something like this? :
"Starting from this release, scikit-image will follow the spirit of the recently introduced numpy deprecation policy -- NEP 29 (https://github.com/numpy/numpy/blob/master/doc/neps/nep-0029-deprecation_policy.rst). Accordingly, scikit-image 0.16 drops the support for Python 3.5 [and numpy __]. (#4102)"